src/collections/members/yash-kamboj/index.mdx

```diff
@@ -0,0 +1,12 @@
+---
+name: Yash Kamboj
+position: Contributor
+image_path: ./yash-kamboj.jpeg
+github: YashKamboj
+twitter: yashkamboj123
+linkedin: Yash Kamboj
+location: New Delhi, India
+bio: I am a undergrad student at IP university, Delhi. I am passionate about open-source and front-end development. Always up to learn new things. Love to help others and learning new thing from them.
+status: Active
+published: true
+---
```

Comment:
@YashKamboj , could you pls confirm the details here!
Thanks

Reply:
@KXLAA change the linkedin to yash-kamboj-9078011b5

Reply:
Everything else, good, Great Work 👌

Reply:
alright